SAMLPortal/Controllers/API/SetupController.cs

```diff
@@ -0,0 +1,79 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Filters;
+using Novell.Directory.Ldap;
+using SAMLPortal.Models;
+
+namespace SAMLPortal.Controllers.API
+{
+	[Route("api/setup")]
+	[ApiController]
+	//[ServiceFilter(typeof(SetupAsyncActionFilter))]
+	//[Produces("application/json")]
+	public class SetupController : Controller
+	{
+		public SetupController()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this redundant constructor.](https://app.codacy.com/manual/zegorax/SAMLPortal/pullRequest?prid=5343548)